package.json

```diff
@@ -9,6 +9,7 @@
     "start": "npm run server",
     "dev": "npm run watch",
     "cypress:open": "cypress open",
+    "cypress:record": "cypress run --record --key --env=cypress-sttm-web",
```

Comment:
check out https://docs.cypress.io/guides/guides/command-line.html#cypress-run-record-key-lt-record-key-gt 

think you just want this to be `cypress run --record` the env variable would be called `CYPRESS_RECORD_KEY`